platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/assertions/MultipleNodeConsensusResultsAssert.java

```diff
@@ -167,6 +157,22 @@ public MultipleNodeConsensusResultsAssert haveMaxDifferenceInLastRoundNum(
         return this;
     }
 
+    /**
+     * Verifies that the created consensus rounds are consistent.
+     *
+     * <p>This includes checking if the ancient thresholds are increasing and the timestamps of
+     * events are strictly increasing.
+     *
+     * @return this assertion object for method chaining
+     */
+    public MultipleNodeConsensusResultsAssert haveConsistentRounds() {
+        isNotNull();
+        for (final SingleNodeConsensusResult result : actual.results()) {
+            OtterAssertions.assertThat(result).hasConsistentRounds();
```

Comment:
nit: Maybe static import?

Reply:
Would be nice, but unfortunately it is not possible because there is an `assertThat()` method defined in `MultipleNodeConsensusResultsAssert`.